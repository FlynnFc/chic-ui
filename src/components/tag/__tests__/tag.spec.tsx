import Tag from '..';
import { render } from 'test/utils';

describe('Tag Component', () => {
  it('Renders Correctly', () => {
    const TagGroup = render(
      <>
        <Tag rounded />
        <Tag type="light" />
        <Tag type="secondary" />
        <Tag type="danger" />
        <Tag type="success" />
        <Tag type="warning" />
        <Tag type="info" />
      </>
    );
    expect(TagGroup).toMatchSnapshot();
  });
});
