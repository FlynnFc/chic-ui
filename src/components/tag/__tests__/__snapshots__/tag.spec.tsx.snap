// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag Component Renders Correctly 1`] = `
Array [
  <span
    className="sc-bdfBQB fJGVPK"
    type="span"
  />,
  <span
    className="sc-bdfBQB eIbEGS"
    type="span"
  />,
  <span
    className="sc-bdfBQB hzBuei"
    type="span"
  />,
  <span
    className="sc-bdfBQB eTRHlG"
    type="span"
  />,
  <span
    className="sc-bdfBQB dzhmLb"
    type="span"
  />,
  <span
    className="sc-bdfBQB cDzqLq"
    type="span"
  />,
  <span
    className="sc-bdfBQB iVxUBU"
    type="span"
  />,
]
`;
